static/app/components/modals/inviteMembersModal/index.spec.tsx

```diff
@@ -234,6 +235,66 @@ describe('InviteMembersModal', function () {
     expect(screen.getByRole('button', {name: 'Send invite'})).toBeDisabled();
   });
 
+  it('sends all successful invites with team default', async function () {
+    jest.mocked(useOrganization).mockReturnValue(org);
+    const createMemberMock = MockApiClient.addMockResponse({
+      url: `/organizations/${org.slug}/members/`,
+      method: 'POST',
+    });
+
+    render(<InviteMembersModal {...modalProps} />);
+
+    await waitFor(() => {
+      expect(screen.getByRole('button', {name: 'Add another'})).toBeInTheDocument();
+    });
```

Comment:
Nit: simplify to `findBy*` selector and use that to enforce async / selection of the element.

```suggestion
    await screen.findByRole('button', {name: 'Add another'})).toBeInTheDocument()
```

Reply:
done everywhere in this file now!